src/chart/Summary.js

```diff
@@ -29,6 +29,10 @@
     Summary.prototype.publish("minHeight", 150, "number", "Minimum Height");
     Summary.prototype.publish("playInterval", null, "number", "Play Interval", null, { optional: true });
 
+    Summary.prototype.publish("summary_format", null, "number", null, null);
```

Comment:
This was introduced in a prior version which had a different approach. It can be removed now that @mounicarachakonda is using the transform